msbuild/Xamarin.Mac.Tasks/Xamarin.Mac.Common.targets

```diff
@@ -453,9 +453,7 @@ Copyright (C) 2014 Xamarin. All rights reserved.
 			RequireCodeSigning="$(EnableCodeSigning)"
 			SdkPlatform="MacOSX"
 			ProvisioningProfile="$(CodeSignProvision)"
-			SigningKey="$(CodeSigningKey)"
-			SdkIsSimulator="false"
-			>
+			SigningKey="$(CodeSigningKey)">
```

Comment:
That's fixing a different issue (or the past PR was incorrect) since `SdkIsSimulator` should never have been present on a `Mac` target file, right ?

2nd Q: is that in `cycle8` ?